ccdproc/core.py

```diff
@@ -580,13 +580,16 @@ def sigma_func(arr):
     ----------
     arr : `~ccdproc.CCDData` or `~numpy.ndarray`
         Array whose deviation is to be calculated.
+        
+    axis : None or int or tuple of ints, optional
+        Axis or axes along which a sum is performed. The default (axis = None) is perform a sum over all the dimensions of the input array. axis may be negative, in which case it counts from the last to the first axis.
 
     Returns
     -------
     float
         standard deviation of array
     """
-    return 1.4826 * stats.median_absolute_deviation(arr)
+    return 1.4826 * stats.median_absolute_deviation(arr, 0)
```

Comment:
``astropy`` uses some more digits for this factor: ``1.482602218505602`` (see http://docs.astropy.org/en/stable/_modules/astropy/stats/funcs.html#mad_std). It may not be necessary but in case ``ccdproc`` gets merged into astropy or ccdproc someday switches to the median the different precision might be problematic.